File: Assignment33_4.c
// Accept character from user and check whether it is in small case or not

#include<stdio.h>
#include<stdbool.h>

bool ChkSmall(char ch)
{
    bool bFlag = false;

    if((ch >= 'a') && (ch <= 'z'))
    {
        bFlag = true;
    }
    else
    {
        bFlag = false;
    }

    return bFlag;
}

int main()

{
    char ch = '\0';
    bool bRet = false;

    printf("Enter character :\n");
    scanf("%s",&ch);                               
    
    bRet = ChkSmall(ch);

    if(bRet == true)
    {
        printf("It is in small case\n");
    }
    else
    {
        printf("It is not small case\n");
    }

    return 0;
}

File: Assignment33_4.java
// Accept character from user and check whether it is small or not

import java.util.*;

class Logic
{
    public boolean CheckAlpha(char ch)
    {
        boolean bFlag = false;

        if((ch >= 'a') && (ch <= 'z'))
        {
            bFlag = true;
        }
        else
        {
            bFlag = false;
        }

        return bFlag;
        
    }
}

class Assignment33_4
{
    public static void main(String[] args) 
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter a character: ");
        char cValue = sobj.next().charAt(0);

        Logic lobj = new Logic();
        boolean bRet = lobj.CheckAlpha(cValue);

        if(bRet == true)
        {
            System.out.println("It is Small");
        }
        else 
        {
            System.out.println("It is not Small");
        }
    }
}